src/clr-angular/forms/combobox/option-items.ts

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (c) 2016-2020 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import { NgForOf, NgForOfContext } from '@angular/common';
+import {
+  Directive,
+  DoCheck,
+  Input,
+  IterableDiffer,
+  IterableDiffers,
+  TemplateRef,
+  TrackByFunction,
+  ViewContainerRef,
+  OnDestroy,
+} from '@angular/core';
+
+import { OptionSelectionService } from './providers/option-selection.service';
+import { Subscription } from 'rxjs';
+import { ClrPopoverPositionService } from '../../utils/popover/providers/popover-position.service';
+
+@Directive({
+  selector: '[clrOptionItems][clrOptionItemsOf]',
+})
+export class ClrOptionItems<T> implements DoCheck, OnDestroy {
+  private iterableProxy: NgForOf<T>;
+  private _rawItems: T[];
+  private filteredItems: T[];
+  private subscriptions: Subscription[] = [];
+  private filter: string;
+  private _filterField: string;
+  private differ: IterableDiffer<T> | null = null;
+
+  @Input('clrOptionItemsOf')
+  public set rawItems(items: T[]) {
+    this._rawItems = items ? items : [];
```

Comment:
same as 

```ts
this._rawItems = items || [];
```

Reply:
I agree, but I find it's a matter of style with no clear advantage of one over the other. What one lacks in bytes, makes up as brevity.